ztp/source-crs/HardwareEventRedfishSecret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
```

Comment:
Including the Secret in ZTP implies that it will be stored in GIT. We should look at how to integrate this feature with external secret managers/vaults.

Reply:
Deleted the file. Since the data needs to be base64 encoded, I think it might be easier for customer just create the secret using literals from CLI. I will update documentation to let customer manually create secret like this:
```
oc -n openshift-hw-events create secret generic redfish-basic-auth --from-literal=username=$BMC_USERNAME --from-literal=password=$BMC_PASSWORD --from-literal=hostaddr="BMC_HOST_IP_ADDR"
```